sla_per_user_system_test.py

```diff
@@ -828,6 +828,12 @@ def _compare_workloads_c_s_metrics(self, workloads_queue: list) -> dict:
         comparison_axis = {"latency 95th percentile": 2.0,
                            "latency 99th percentile": 2.0,
                            "op rate": 2.0}
+        release = parse_version(self.db_cluster.nodes[0].scylla_version.replace("~", "-")).release
```

Comment:
The following existing SCT utility should have been used for the Scylla version comparisons:
https://github.com/scylladb/scylla-cluster-tests/blob/318be432/sdcm/utils/version_utils.py#L123

Example of usage: https://github.com/scylladb/scylla-cluster-tests/blob/318be432/sdcm/utils/sstable/sstable_utils.py#L72-L80